docs/features/search/README.md

```diff
@@ -0,0 +1,100 @@
+---
+id: search-overview
+title: Search Documentation
+sidebar_label: Overview
+description:
+  Backstage Global Search lets you find the right information you are looking
+  for in the Backstage ecosystem.
+---
+
+# Backstage Global Search
```

Comment:
Should we call it `Backstage Global Search` or just `Backstage Search`. Thinking in terms of a new user discovering Backstage for the first time.

Reply:
good question, i've mixed them both 😆, but yes we should go for one. Let's go for Backstage Search to avoid confusion.